pennylane/ops/qubit.py

```diff
@@ -2157,6 +2157,16 @@ def _matrix(cls, *params):
 
         return U
 
+    @staticmethod
+    def decomposition(U, wires):
+        # Decompose arbitrary single-qubit unitaries as the form RZ RY RZ
+        if qml.math.shape(U)[0] == 2:
+            wire = Wires(wires)[0]
+            decomp_ops = qml.transforms.decompositions._zyz_decomposition(U, wire)
+            return decomp_ops
+
+        return NotImplementedError("Decompositions only supported for single-qubit unitaries")
```

Comment:
Yes, thanks for the suggestion to separate it out! :grin: It will also make it much cleaner to add different decompositions, one for 2-qubit unitaries, etc. while keeping the qubit ops file tidy.